src/styles/blocks/card.css

```
@@ -0,0 +1,71 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__product-image {
+  display: block;
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__product-name {
+  font-weight: var(--font-medium);
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  color: var(--secondary-color);
+  margin-bottom: 16px;
+}
+
+.card__product-rating {
+  margin-bottom: 24px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-value {
+  font-size: 12px;
+  color: var(--secondary-color);
+}
+
+.card__value {
+  font-size: 16px;
+  font-weight: var(--font-bold);
+}
+
+.card__buy-button {
+  display: block;
+  padding: 10.5px 0;
```

don't use fractional values, that probably means that you did smth wrong
```suggestion
  padding: 11px 0;
```

or

```suggestion
  padding: 10px 0;
```

---

But, IMO, you can set height here and center your text using flex